README.md

```diff
@@ -77,88 +64,93 @@ npm init nx-workspace myworkspace
 yarn create nx-workspace myworkspace
 ```
 
-## Adding Nx to an Existing Angular CLI workspace
+### Adding Nx to an Existing Angular CLI workspace
```

Comment:
```suggestion
## Adding Nx to an Existing Angular CLI workspace
```

I think adding to an existing angular is separate from creating a workspace

Reply:
I think the intent is the same: create an nx workspace. I just want to say "you could also add it instead of creating a new workspace"

Reply:
Yeah, I think it falls under the broader scope of getting started as 2 separate options.